python/swe/benchmark/constants.py

```diff
@@ -7,3 +7,5 @@
 
 DATASET_ON_DISK = "./swe_lite_dataset"
 TEST_SPLIT = "[1:300]"
+
+DEFAULT_IMAGE_NAME = "sweagent/swe-agent"
```

Comment:
Need to make `techcomposio/swe-agent` Check the spelling from docker. I uploaded this one yday.